src/fastify.js

```diff
@@ -0,0 +1,33 @@
+import fastifyServer from 'fastify';
+
+class Fastify {
```

Comment:
Too specific name. In case we want to replace `fastify` with something else we will have a lot of changes in the application